### PR TITLE
chore: project addition by ratishjain12 #1782

### PR DESCRIPTION
**Pull request by ratishjain12**
I have added my projects and profile please review and merge it 

fixes issue #1782

### CODE REVIEW

File: DB/projects.json [hunk 1]
```diff
@@ -5400,11 +4153,42 @@
...
+    "github_username": "ratishjain12",
+    "Social_media": {
+      "gitHub": "https://github.com/ratishjain12",
+      "LinkedIn": "https://www.linkedin.com/in/ratish-jain/",
+      "Instagram": "https://www.instagram.com/ratish_45/",
+      "Twitter": "https://twitter.com/ratishtwts"
+    },
+    "Projects": [
+      {
+        "link": "https://discord-clone-sepia.vercel.app/",
```
Comment (priyankarpal) [inline DB/projects.json:4170]: add the github repo link

File: DB/projects.json [hunk 2]
```diff
@@ -5400,11 +4153,42 @@
...
+        "title": "Discord Clone",
+        "description": "A Fully Responsive Discord Landing Page UI Clone with animations ",
+        "tech": ["html", "tailwindcss", "javascript", "GSAP"]
+      },
+      {
+        "link": "https://blogify-mern-app.vercel.app/",
```
Comment (priyankarpal) [inline DB/projects.json:4176]: add the github repo link

File: DB/projects.json [hunk 3]
```diff
@@ -5400,11 +4153,42 @@
...
+      "LinkedIn": "https://www.linkedin.com/in/ratish-jain/",
+      "Instagram": "https://www.instagram.com/ratish_45/",
+      "Twitter": "https://twitter.com/ratishtwts"
+    },
+    "Projects": [
+      {
+        "link": "https://discord-clone-sepia.vercel.app/",
+        "title": "Discord Clone",
+        "description": "A Fully Responsive Discord Landing Page UI Clone with animations ",
+        "tech": ["html", "tailwindcss", "javascript", "GSAP"]
```
Comment (priyankarpal) [inline DB/projects.json:4173]: change to lowercase 
```suggestion
        "tech": ["html", "tailwindcss", "javascript", "gsap"]
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]
@ratishjain12 also, add the issue number to the PR description

File: DB/projects.json (hunk 1 above)
Comment (ratishjain12) [reply DB/projects.json]: I have resolved this

File: DB/projects.json (hunk 2 above)
Comment (ratishjain12) [reply DB/projects.json]: I have resolved this

File: DB/projects.json (hunk 3 above)
Comment (ratishjain12) [reply DB/projects.json]: I have resolved this

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**